test/spinbox-test.js

```diff
@@ -285,6 +285,21 @@ define( function ( require ) {
 
 	} );
 
+	QUnit.test('spinbox should trigger change after using setValue', function (assert) {
+		var $spinbox = $(html).find('#MySpinboxDecimal').spinbox({
+			value: '1'
+		});
+
+		$spinbox.spinbox('setValue', '2');
+
+		$spinbox.on('changed.fu.spinbox', function () {
+			assert.ok(true, 'spinbox triggers changed after input' );
```

Comment:
this is fine for now but for future reference when using an event listener an async test is more appropriately because it is unknown what is happening before the event is triggered.